.xperior/testds/motr-single_tests.yaml

```diff
@@ -434,17 +435,17 @@ Tests:
 
 #   MOTR-2819
   - id       : 41motr-conf-update
-    script   : motr_conf_update_st.sh
-    dir      : src/motr/st/utils/
+    script   : 'm0 run-st 41motr-conf-update'
+    dir      : src/scripts/
     executor : Xperior::Executor::MotrTest
     sandbox  : /var/motr/sandbox.41motr-conf-update
     groupname: 02motr-single-node
     polltime : 30
     timeout  : 1800
 
   - id       : 42motr-utils
-    script   : motr_utils_st.sh
-    dir      : src/motr/st/utils/
+    script   : 'm0 run-st 42motr-utils'
+    dir      : src/scripts
```

Comment:
Please note,
is the ending slash "/" needed here?

Reply:
Removed